FlowCryptAppTests/ExtensionTests.swift

```diff
@@ -94,7 +94,7 @@ extension ExtensionTests {
         let otherYearDate = Date(timeIntervalSince1970: 1579883652)
 
         XCTAssertTrue(dateFormatter.date(from: DateFormatter().formatDate(sameDayDate)) != nil)
-        XCTAssertEqual(DateFormatter().formatDate(sameYearDate), "Jan 24")
+        XCTAssertEqual(DateFormatter().formatDate(sameYearDate), "00:00")
```

Comment:
This case was wrong because for the same year and month we format only time

Reply:
OK, let's wait for your fix

Reply:
This test was fixed by another PR - https://github.com/FlowCrypt/flowcrypt-ios/blob/master/FlowCryptAppTests/ExtensionTests.swift#L100, so it should pass after merging `master`